var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -247,8 +247,11 @@ def configure_args(self):
             static_bootstrap_flags = '-static-libstdc++ -static-libgcc'
             binutils_options = [
                 '--with-sysroot=/',
-                '--with-stage1-ldflags={0} {1}'.format(
-                    self.rpath_args, static_bootstrap_flags),
+                # The following line causes linkage error on RHEL7
```

Comment:
@cedricchevalier19 You removed the line, but not this comment that is intended to refer to that line.